demos/global-matrix-auth/README.md

```diff
@@ -7,12 +7,8 @@ jenkins:
   authorizationStrategy:
     globalMatrix:
       grantedPermissions:
-        - name: "anonymous"
-          permissions:
-            - "Overall/Read"
-        - name: "authenticated"
-          permissions:
-            - "Overall/Administer"
+        - "Overall/Read:anonymous"
```

Comment:
I don't the the quoting is needed but I guess it is a style preference 😅

Reply:
@casz This is mine atm:
```yaml
  authorizationStrategy:
    globalMatrix:
      grantedPermissions:
        # Necessary for logged-in users to view and run jobs:
        - "Job/Read:authenticated"
        - "View/Read:authenticated"
        - "Overall/Read:authenticated"
        - "Job/Cancel:authenticated"
        - "Job/Build:authenticated"
        # Necessary for admin view (which also includes config):
        - "Overall/Administer:x10an14"
```